python-test/features/steps/agentsProvider.py

```diff
@@ -0,0 +1,54 @@
+from behave import given, when, then
+import time
+import functions
+import os
+import docker
+
+email = os.getenv('EMAIL')
+password = os.getenv('PASSWORD')
+
+agentname = os.getenv('AGENT_NAME')
```

Comment:
Let's have a default name for the agent (maybe a random name?).
When running the tests, we must provide flexibility (like having this env variable) but we should also hide as much complexity as we can.